docs/developing/how-to-review-a-pull-request.rst

```diff
@@ -0,0 +1,94 @@
+============================
+How to review a pull request
+============================
+
+This is our shared, collaborative how-to for reviewing a pull request at
+Hypothesis. Like everything else in our documentation, this isn't set in stone
+but is a living, collaborative document. If you want to question or change
+something in this guide, open a GitHub issue or pull request.
+
+
+-------------------
+Aims of code review
+-------------------
+
+What we aim to achieve by code review at Hypothesis:
+
+* Foster and enact a positive, patient and friendly internal culture for our
+  team
+* Improve working relationships between developers
+* Relieve tension by giving positive feedback
+* Learn from the code that you're reviewing
+* Break down code silos, giving each member a broad exposure to different parts
+  of the codebase
+* Mentor and educate
+* Find bugs and improve code quality
+
+------
+Don'ts
+------
+
+* Don't be blunt and just point out the things that are wrong and that you want
+  changed, without saying anything else.
+
+* Don't harshly criticize code, and don't use personal tone, hyperbole, or
+  demanding, challenging, insulting, impatient or passive-aggressive language
+  when criticizing code.
+
+* Don't be a back-seat coder. Consider your suggestions carefully and pick the
+  best ones.
+
+* Don't break the rule of no surprises. Suggestions should be evidence-based
+  not opinion-based. For example, based on our
+  :doc:`documented coding standards <code-style>` rather than on conflicting
```

Comment:
Sadly there's not much to these coding standards. They're a woefully incomplete, dead and ignored document rather than a living shared agreement among the team. Could be a lot more here